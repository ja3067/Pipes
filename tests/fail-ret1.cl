def foo() -> int:
	if True:
		return 3.0
	else:
		return 4

print(foo())
