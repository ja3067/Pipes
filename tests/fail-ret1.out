RuntimeError: invalid return type (expected int)
